content/quickstart/go/metrics.md

```diff
@@ -406,15 +401,17 @@ Now we will insert a specific tag called "repl". It will give us a new `context.
 
 For example
 ```go
-        ctx, err := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"))
+ctx, _ := tag.New(context.Background(), tag.Insert(KeyMethod, "repl"),
```

Comment:
Please run these Go snippets through go-fmt first.

Reply:
thank you!